specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-06-01/Encoding.json

```diff
@@ -2296,6 +2323,9 @@
           "items": {
             "$ref": "#/definitions/JobOutput"
           },
+          "x-ms-identifiers": [
+            "label"
```

Comment:
label in the definition and a few other places, is not a required property. does it make sense to keep the array empty here?

Reply:
Yeah totally. If there is no property can be used as an identifier, we can keep the x-ms-identifiers array empty.

Reply:
I have pushed a new commit for removing the not required label property from xms-ids. Please take a look.

Reply:
Is this change needed urgently? all our swaggers are auto generated from code. If we make this change now directly to swagger, in the next release, I have to manually manage these. I rather they be added in our service code, so they are added properly.

Another question, is changing these identifiers considered breaking change?

Reply:
The check for x-ms-identifiers will be turned on after the whole initial work is done. I am sorry that this may add more work to you. @yantang-msft this could be a generic problem for teams that auto-generate their swaggers. What would be your suggestions on this?

AFAIK, changing these identifiers in the future should not be a breaking change.

Reply:
Hi @giakas, I checked with my team and unfortunately you team would probably have to manually add these annotations to the autogenerated swagger next time you generate them, or update the code generation tool to include this new annotation, which are what some other teams that have autogenerated swagger will be doing. I think for now we can check in this change first if you don't have any more code change request, which eliminates any unexpected error around x-ms-identifiers of your swaggers, and if you have any concerns or suggestions about how to make the work 
 better, please do feel free to reach out to me at ltong at microsoft.com or my team changeanalysisteam at microsoft.com, we would be more than happier to listen and chat about it over there.